test/functional/device/accessibility-e2e-specs.js

```diff
@@ -13,7 +13,7 @@ describe('Accessibility', function() {
   let driver, caps;
 
   beforeEach(function() {
-    caps = amendCapabilities(SETTINGS_CAPS, { 'appium:usePrebuiltWDA': true });
+    caps = amendCapabilities(SETTINGS_CAPS);
```

Comment:
I assume we should use some more advanced logic there, e.g. don't add this capability in the first test and then add it as soon as a driver session successfully starts at at least once. Removing it completely may significantly slow down suite performance

Reply:
Yeah, I'll try it